### PR TITLE
feat(design): remove error message dependency on form-field

### PR DESCRIPTION
**Pull request by xelaint**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
`daff-error-message` depends on `daff-form-field`. It cannot be used if we create a `daff-solo-field` component.

Part of: #416 

## What is the new behavior?
Move `daff-error-message` outside of `daff-form-field` and create its module so that it can be used in other form components.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: libs/design/src/atoms/form/error-message/error-message.component.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+import { Component, ViewEncapsulation, HostBinding } from '@angular/core';
 
 @Component({
   selector: 'daff-error-message',
   template: '<ng-content></ng-content>',
   styleUrls: ['./error-message.component.scss'],
-  host: {
-    'class': 'daff-error-message'
-  },
   encapsulation: ViewEncapsulation.None
```
Comment (damienwebdev) [inline libs/design/src/atoms/form/error-message/error-message.component.ts]: Can this be encapsulated?

File: libs/design/src/atoms/form/error-message/error-message.module.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { DaffErrorMessageComponent } from './error-message.component';
+
+@NgModule({
+  imports: [
+    CommonModule
```
Comment (damienwebdev) [inline libs/design/src/atoms/form/error-message/error-message.module.ts]: I don't know if we need the common module.

File: libs/design/src/atoms/form/error-message/error-message.component.ts (hunk 1 above)
Comment (xelaint) [reply libs/design/src/atoms/form/error-message/error-message.component.ts]: Yes, updated.

File: libs/design/src/atoms/form/error-message/error-message.module.ts (hunk 2 above)
Comment (xelaint) [reply libs/design/src/atoms/form/error-message/error-message.module.ts]: Removed

**Pull request merged**